pool_automation/roles/aws_manage/tasks/set_known_hosts.yml

```diff
@@ -0,0 +1,36 @@
+---
+- name: Check in-variables
+  assert:
+    that:
+      - lookup('vars', item, default='')
+    msg: "{{ lookup('vars', item, default='undefined')|string }}"
+  loop:
+    - known_hosts_file
+    - hosts
+
+- name: Ensure '{{ known_hosts_file }}' exists
+  copy:
+    content: ""
+    dest: "{{ known_hosts_file }}"
+    force: no
+
+- name: Check existent records in '{{ known_hosts_file }}' file
```

Comment:
`known_hosts` ansible module is idempotent, so it's not really needed to scan existing known_hosts to avoid double adding keys. Is it done for performance reasons?

Reply:
yes, it is just to not to perform any unnecessary scans if the host in known_hosts already